mamba/_version.py

```diff
@@ -1,3 +1,2 @@
-# -*- coding: utf-8 -*-
 version_info = (0, 4, 4)
-__version__ = '.'.join(map(str, version_info))
+__version__ = ".".join(map(str, version_info))
```

Comment:
it would be nice to change this part and add ``setuptools_scm``
I can do it in another PR if you agree to that

Reply:
Can we keep the version_info tuple though? We parse it in CMake to put the same version number into the mamba_lib / micromamba binary here: https://github.com/TheSnakePit/mamba/blob/5e801e3ecebf98b1c3655938eee051850dcfb7b0/CMakeLists.txt#L51-L60

Reply:
Sure. I will change it, no problem
For a new PR I was thinking to add ``setuptools_scm`` and get the version at runtime calling the python function from the C++ code. As ``pybind11`` is already a dependency that should be simple to do.
What do you think?